### DISCUSSION TITLE
How to get the Rollup Field on Parent Object to return "0" as default

### DISCUSSION THREAD

**Discussion by baobao917** [jamessimone/apex-rollup#490] [Q&A] [answered]
I'm trying to create rollups to summarize the Aging of invoices.  It works but if there don't happen to be any invoices in a particular aging bucket, the rollup field on the parent object is null instead of showing as "0". 

![image](https://github.com/jamessimone/apex-rollup/assets/104730171/09a100a5-0b51-4a6d-80b8-b76a7786bea6)
![image](https://github.com/jamessimone/apex-rollup/assets/104730171/0d8e8edb-28fa-44f2-999f-800249852ddc)

I saw this piece of instruction that seems to speak to what I'm trying to accomplish
![image](https://github.com/jamessimone/apex-rollup/assets/104730171/fbdbea6d-d490-4716-bd80-3a782b45f76b)

However, I tried that and it doesn't appear to return a 0 value.  I tried it with "Is Full Record Set" = TRUE as well but that also didn't work.  Is what I'm trying to do possible or am I doing it incorrectly?
![image](https://github.com/jamessimone/apex-rollup/assets/104730171/f0e8a498-d607-42a9-a98d-5c042ebc4448)

**Comment by jamessimone** [comment]
@baobao917 I will take a look, this should be an easy fix

**Reply by baobao917** [reply]
@jamessimone I wasn't sure if this fix was part of one of the recent releases.  Wasn't sure if I should go ahead and mark this as answered/resolved.

**Reply by jamessimone** [reply, marked as answer]
@baobao917 are you seeing 0 getting applied properly now? I know I did tweak this - or at least start to look at it a few weeks ago - but unfortunately I don't quite remember where we ended up on this one.

**Reply by baobao917** [reply]
@jamessimone Yes, it works now.

**Reply by jamessimone** [reply]
Awesome, great to hear
